.github/workflows/ci.yml

```diff
@@ -93,10 +93,24 @@ jobs:
         branch: create-pull-request/l10n
         delete-branch: true
 
-  snap:
+  ubuntu-dektop-installer-snap:
```

Comment:
could we move this to `installer.yaml` where the installer specific jobs are, and add `welcome.yaml` for welcome-specific jobs?